decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -298,6 +298,56 @@ def minutes_data?
         [video_url, audio_url].any?(&:present?)
       end
 
+      def self.sort_by_translated_title_asc
+        order(Arel::Nodes::InfixOperation.new("",
+                                              Arel::Nodes::InfixOperation.new("->>", arel_table[:title], Arel::Nodes.build_quoted(I18n.locale)), Arel.sql("ASC")))
+      end
+
+      def self.sort_by_translated_title_desc
+        order(Arel::Nodes::InfixOperation.new("",
+                                              Arel::Nodes::InfixOperation.new("->>", arel_table[:title], Arel::Nodes.build_quoted(I18n.locale)), Arel.sql("ASC")))
+      end
+
+      ransacker :type do
+        Arel.sql(%("decidim_meetings_meetings"."type_of_meeting"))
+      end
+
+      ransacker :title do
+        Arel.sql(%{cast("decidim_meetings_meetings"."title" as text)})
+      end
+
+      ransacker :id_string do
+        Arel.sql(%{cast("decidim_meetings_meetings"."id" as text)})
+      end
+
+      ransacker :is_upcomming do
+        query = <<-SQL.squish
+        (
+          SELECT EXISTS (
+            SELECT 1 FROM decidim_meetings_meetings as dmm WHERE dmm.id = decidim_meetings_meetings.id AND dmm.start_time > NOW()
+          )
+        )
+        SQL
+        Arel.sql(query)
```

Comment:
💬 Can we use `Arel.sql("start_time > NOW()")` here?

Reply:
Replacing that block with `Arel.sql("start_time > NOW()")` doesn't seems to work

Reply:
@leio10 The boolean filter is_upcoming is built the same way the "is_emendation" is built on Proposal: 
https://github.com/tremend-cofe/decidim/blob/fix/7399/decidim-proposals/app/models/decidim/proposals/proposal.rb#L358

Reply:
@roxanaopr Sorry, my suggestion was not working because of the lack of parenthesis. It works if you write `Arel.sql("(start_time > NOW())")`. 

With that, when the filter is `is_upcoming_true: true` it builds the query `SELECT "decidim_meetings_meetings".* FROM "decidim_meetings_meetings" WHERE (start_time > NOW()) = TRUE`, that gets all the meetings with the `start_time` greater than `NOW()`. 

It also works with `is_upcoming_true: false`, `is_upcoming_false: true`, `is_upcoming_false: false` and even with `is_upcoming_eq: true`.

@alecslupu in the example you are referring, the subquery it's needed because you are checking some fields that are not in the model's table. But in this case the condition is based on a field included in the model's table. So, you can just filter by that condition without any subquery, and the result is a simpler and faster query.

Reply:
@leio10 I have performed the requested `Arel.sql("(start_time > NOW())")` change